test/libsolidity/syntaxTests/parsing/empty_tuple_assignment.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
```

Comment:
Oh and please don't call this `empty_tuple_assignment` - an empty tuple is `()`. Perhaps `destructuring_assignment_no_variable`?

Reply:
`destructuring` sounds reasonable. thx.